service/expression.js

```diff
@@ -26,8 +27,13 @@ class ExpressionService {
   }
 
   async removeExpression(expressionId) {
-    await expressionRepository.deleteExpression(expressionId);
-    await expressionRepository.deleteExpressionFromUsers(expressionId);
+    const deleted = await expressionRepository.deleteExpression(expressionId);
+
+    if (!deleted) {
+      throw new InexistentItem('ExpressionId does not exists.');
```

Comment:
This message is quite confusing. I think you are talking about an expression with that id that doesn't exists.